### PR TITLE
Compatibility fixes for BF 3.3

### PR DESCRIPTION
**Pull request by basdelfos**
Blackbox_decode fixes:
- Vbat and Amp values (bb logs now contain the converted ADC values instead of raw sensor values).
- Synced flight and status enums with BF.
- Retarget to VS2017 platform tools.

### CODE REVIEW

File: visual-studio/.gitignore [hunk 1]
```diff
@@ -1 +1,4 @@
 /Baseflight blackbox tools.v12.suo
+
+# artefacts for Visual Studio Code
```
[the remark this answers is not in the document]
Comment (basdelfos) [reply visual-studio/.gitignore]: Ah this is why the `.vs` files are included in the commit (didn't check it before submitting tho).
There is a second `.gitignore` in the `visual-studio` folder. It contains part of the changes I made, think that VS2017 copied it over from the root to the folder where the `.sln` files exists.

> I prefer `artifacts` like in Jenkins and TeamCity, but that's ok.

What do you mean exactly?

[the remark this answers is not in the document]
Comment (DieHertz) [reply visual-studio/.gitignore]: Spelling

**Comment by basdelfos** [comment]
All requested changes are done and squased to one commit

File: visual-studio/blackbox_decode/blackbox_decode.vcxproj [hunk 2]
```diff
@@ -1,5 +1,5 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
```
[the remark this answers is not in the document]
Comment (mikeller) [reply visual-studio/blackbox_decode/blackbox_decode.vcxproj]: Would it be an option to **not** drop the VS support (for existing users), and just add CMake support as well?

**Comment by McGiverGim** [comment]
I think that this PR had been forgotten 😉

**Comment by basdelfos** [comment]
Hmm, I had the assumption I already fixed mikeller review comments. But apparently not. Will look into this in one of the coming days.

**Comment by mikeller** [comment]
Sorry, my bad, I forgot about this.

**Pull request merged**